### PR TITLE
Add animations to the UI all over

### PR DESCRIPTION
**Pull request by BruceJi7**

### CODE REVIEW

File: src/pages/Category/index.tsx [hunk 1]
```diff
@@ -64,28 +67,38 @@ function Category() {
...
-              </div>
+              </motion.div>
             ))}
           </React.Fragment>
         ))}
-      </div>
-      {isLoading && <Spinner />}
+      </motion.div>
+      <Waypoint bottomOffset="-20%" onEnter={() => fetchNextPage()} />
       {!isLoading && !files?.pages?.[0]?.data && (
```
Comment (ivanms1) [inline src/pages/Category/index.tsx:99]: you can remove `isLoading`

File: src/pages/FileDetails/index.tsx [hunk 2]
```diff
@@ -16,6 +17,8 @@ import BackIcon from "../../assets/icons/left-arrow.svg";
 
 import styles from "./FileDetails.module.scss";
 
+import { filePageAnimation } from "../../anim/";
```
Comment (ivanms1) [inline src/pages/FileDetails/index.tsx:20]: move above styles

File: src/pages/Search/index.tsx [hunk 3]
```diff
@@ -8,6 +9,7 @@ import Spinner from "../../components/Spinner";
 
 import useUrlQuery from "../../hooks/useUrlQuery";
 
+import { categoryAnimation, cellEntryAnim } from "../../anim";
```
Comment (ivanms1) [inline src/pages/Search/index.tsx:12]: please add a space to separate type of imports

File: src/pages/Search/index.tsx [hunk 4]
```diff
@@ -60,21 +62,33 @@ function Search() {
...
-              </div>
+              </motion.div>
             ))}
           </React.Fragment>
         ))}
-      </div>
-      {isLoading && <Spinner />}
+      </motion.div>
+
       {!isLoading && !files?.pages?.[0]?.data && (
```
Comment (ivanms1) [inline src/pages/Search/index.tsx:92]: `isLoading` can be removed

File: src/pages/UserFiles/Downloads/index.tsx [hunk 5]
```diff
@@ -14,6 +15,7 @@ import BackIcon from "../../../assets/icons/back.svg";
 import { FileProps } from "../../../types";
 
 import styles from "./Downloads.module.scss";
+import { mainPageAnimation } from "../../../anim";
```
Comment (ivanms1) [inline src/pages/UserFiles/Downloads/index.tsx:18]: import order

**Review by ivanms1 — APPROVED** [review]
just some small changes

File: src/pages/Category/index.tsx (hunk 1 above)
Comment (BruceJi7) [reply src/pages/Category/index.tsx:99]: That's like 99, right?

**Pull request merged**